src/json.hpp

```diff
@@ -6564,7 +6578,7 @@ class serializer
                         // check that the additional bytes are present
                         assert(i + bytes < s.size());
 
-                        // to use \uxxxx escaping, we first need to calculate
+                        // to use \uxxxx escaping, we first need to caluclate
```

Comment:
Typo introduced here.

Reply:
fixed the typo . 
https://github.com/nlohmann/json/pull/890/commits/a9e1cbb9a574276dca487ccfcf7047adda5db56e